easyprivacy/easyprivacy_thirdparty.txt

```diff
@@ -1234,6 +1234,7 @@
 ||metrics.mdstrm.com^
 ||metrics.pico.tools^
 ||metrics.readme.io^
+||metrics.redgifs.com^
```

Comment:
<details><summary>note to self (@spirillen)</summary>

[note](https://mypdns.org/my-privacy-dns/matrix/-/issues/1163204)

</details>